basic/http/collect-user.js

```diff
@@ -0,0 +1,47 @@
+const Koa = require('koa');
+const Router = require('koa-router');
+
+const app = new Koa(),
+ router =  new Router();
+
+var collect = {"data": 
+{    
+"users":
+  [{
+    "id":"1",
+    "name":"the name",
+    "email":"thename@gmail.com",
+    "password":"random string"
+  }]
+}
+};
```

Comment:
Convert this to class:
```
class User {
  data = [{ ... }];
  
  get() {
    this.data;
  }
}
```

Later on we will add a method for pushing an object inside the `data`

Reply:
RESOLVED 💯